.github/workflows/_build_macos.yml

```
@@ -111,7 +111,7 @@ jobs:
         run: |
           cd build
           conan info --lockfile=lockfile.json ../${{ inputs.conanfile }} --json=project.json
-          conan install --lockfile=lockfile.json ../${{ inputs.conanfile }} --build=missing --generator=virtualenv
+          conan install --lockfile=lockfile.json ../${{ inputs.conanfile }} --build=missing --update
```

```suggestion
          conan install --lockfile=lockfile.json ../${{ inputs.conanfile }} --build=missing
```

Here you don't want to update the lockfile, it might modify the revisions that were already locked.